examples/express-proxy/src/server.ts

```diff
@@ -0,0 +1,40 @@
+import "dotenv/config";
+import { dirname, join } from "node:path";
+import { fileURLToPath } from "node:url";
+import { handler, route } from "@decartai/proxy/express";
+import express from "express";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+const app = express();
+
+// Serve static files (HTML, JS, CSS)
+app.use(express.static("public"));
+
+// Serve SDK from node_modules for the example
+const sdkDistPath = join(__dirname, "../../../packages/sdk/dist");
+console.log(`Serving SDK from: ${sdkDistPath}`);
+app.use(
+  "/node_modules/@decartai/sdk",
+  express.static(sdkDistPath, {
+    setHeaders: (res, path) => {
+      // Set proper content type for JavaScript modules
+      if (path.endsWith(".js")) {
+        res.setHeader("Content-Type", "application/javascript");
+      }
+    },
+  }),
```

Comment:
probably best to import the sdk regularly, and use pnpm workspaces to link it to the local version?
also - what do we need this for?

Reply:
we need this to [access the sdk from the example](https://github.com/DecartAI/sdk/blob/api-572-sdk-proxy-server/examples/express-proxy/public/index.html#L197) html page.